documentation/index.rst

```diff
@@ -179,8 +179,8 @@ The platform operator of FlexMeasures can be an Aggregator.
     :caption: Tutorials
     :maxdepth: 1
     
-    tut/toy-example-from-scratch
     tut/installation
+    tut/toy-example-from-scratch
```

Comment:
I think this was listed before the installation as the tutorial is meant to be possible without actually installing FlexMeasures.

Reply:
But the Toy example already has steps for installing `Flexmeasures` on PC using `pip`. Unless, the objective is to exclusively run flexmeasures via docker before user can actually install it on their PC.

Reply:
The objective is rather that they don't think they *have to* install to give it a little test drive.